### PR TITLE
#3278 Limit input for user could not submit lower than minimum offer …

### LINKED ISSUE #3278: Limit input for user could not submit lower than minimum offer size
As we would like to avoid submitting lower than allowed offer size, 
limit users submitting lower offers 
This would prevent users not waiting for results from rpc   

https://beta.kodadot.xyz/bsx/gallery/2735277589-3?page=1

image

### PR DESCRIPTION
**Pull request by prachi00**
…size

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #3278
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-06-30 at 11 21 39 PM

### CODE REVIEW

File: components/bsx/Gallery/Item/AvailableActions.vue [hunk 1]
```diff
@@ -98,6 +109,28 @@ export default class AvailableActions extends mixins(
...
+        !this.isMakeOffersAllowed || this.minimumOfferAmount > this.balance
+
+      if (!this.isMakeOffersAllowed) {
+        this.tooltipOfferLabel = this.$t('tooltip.makeOfferDisabled')
+      } else if (this.minimumOfferAmount > this.balance) {
+        this.tooltipOfferLabel = this.$t('tooltip.makeOfferNotEnoughBalance')
+      }
+    }, 3000)
+  }
+
```
Comment (vikiival) [inline components/bsx/Gallery/Item/AvailableActions.vue:133]: May I ask Why are you using set timeout ?

Comment (vikiival) [reply components/bsx/Gallery/Item/AvailableActions.vue:133]: There is onApiConnected hook

Comment (prachi00) [reply components/bsx/Gallery/Item/AvailableActions.vue:133]: oh didnt know about the hook, will use that thanks

**Review by kkukelka — APPROVED** [review]
looks good now

**Comment by kkukelka** [comment]
pay 100

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 48.1 USD/KSM ~ 2.079 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xc3145ee6b7397cc6c751785ab526b9a56080088e62578a391478a76be460eace](https://kusama.subscan.io/extrinsic/0xc3145ee6b7397cc6c751785ab526b9a56080088e62578a391478a76be460eace)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
hey seems I was able to submit offer for 90 BSX 

image

**Pull request merged**